packages/flutter/lib/src/material/colors.dart

```diff
@@ -331,6 +331,18 @@ class Colors {
   ///    but with different opacities.
   static const Color white30 = const Color(0x4DFFFFFF);
 
+  /// White with 24% opacity.
+  ///
+  /// ![](https://flutter.github.io/assets-for-api-docs/material/Colors.whites.png)
```

Comment:
please update the diagram in the assets repo :-) (should be very easy to update)

Reply:
I will do so, as soon as this PR lands.